### PR TITLE
add variable

### PR DESCRIPTION
**Pull request by kurkenov**
@geoffrey1330 Can you pls review?

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+variable env_code {
+  type = string
+}
```
Comment (geoffrey1330) [inline variables.tf:3]: could you please Add newline at the end of file

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (kurkenov) [reply variables.tf:3]: Yeah sure

File: terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1 @@
+env_code = almaty
```
Comment (geoffrey1330) [inline terraform.tfvars:1]: could you please add newline at the end of file

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.terraform
+terraform.tfstate
+terraform.tfstate.backup
+.terraform.lock.hcl
+.DS_store
+.terraform.tfstate.lock.info
+.terraform*
```
Comment (geoffrey1330) [inline .gitignore:7]: Add newline at the end of file

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**